### DISCUSSION TITLE
Mp3s or rather free .ogg Music preloading

### DISCUSSION THREAD

**Discussion by GamesStudioTwo** [axmolengine/axmol#2033] [General]
Hi all  everything going well.   quick ? about background music is the any difference if I preload() background music.mp3
rather then just play2p(myfile.mp3) I'm just looking at saving memory, if I don't preload dose it just play from drive and don't store it in memory ? as I believe .mp3 are run time compression ? or will it decompress the lot anyway if I preload or not ?
I'm preloading all the short sounds so they fire on que.
Many thanks in advance.

**Comment by paulocoutinhox** [comment]
Hi,

In my case, i use mp3 file for background music when ame starts.

And on each stage i preload mp3 files for that stage.

Thanks.

**Reply by GamesStudioTwo** [reply]
I'm doing that anyway but do you know if it save memory if I don't preload the music track?

**Reply by paulocoutinhox** [reply]
Yes, preload mean that it will be ready for play, and it will cost memory.
If you don't preload, you will have a delay when you play de sound, because it will need be loaded at runtime.

**Reply by GamesStudioTwo** [reply]
oh that's fine I wont preload the music track that's like 3 mins long, I will just preload the short wav files that less than a second or 2
